singers/index.js

```diff
@@ -0,0 +1,30 @@
+function fetchUserData() {
+
+
+    fetch('./data/singers.json')
+        .then(response => response.json())
+            .then(data => {
+                console.log(data);
+                const mainDiv = document.getElementById("main");
+                console.log(mainDiv);
+
+                const ul = document.createElement('ul');
```

Comment:
First, write the JavaScript logic only and print in the output in the console..  No need to print anything to HTML now.